Makefile

```diff
@@ -29,7 +29,7 @@ define CHRONOGIRAFFE
           ,""      _\_
         ,"  ## |   0 0.
       ," ##   ,-\__    `.
-    ,"       /     `--._;) - "HAI, I'm Chronogiraffe.  Will you be my friend?"
+    ,"       /     `--._;) - "Do ya feel lucky, punk? Huh, do ya?"
   ,"     ## /
```

Comment:
Asked @cryptoquick to do this PR so that you wouldn't push back on this one 😉  Changed it 😸